apps/ledger-live-mobile/src/components/TranslatedError/hooks/useErrorLinks.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useCallback, useMemo } from "react";
+import { Linking, Text } from "react-native";
+
+export function useErrorLinks(error?: Error | null) {
+  const onErrorLinkPress = useCallback(async (errorLink: string) => {
+    if (errorLink.startsWith("http")) {
+      const canOpenURL = await Linking.canOpenURL(errorLink);
+
+      if (canOpenURL) {
+        Linking.openURL(errorLink);
+        return;
+      }
+    }
+
+    if (errorLink === "/platform/multibuy") {
```

Comment:
Should we also take into account the possibility for `/discover/multibuy`

Also there is a good chance that soon the multibuy app will use its own route, not sure if this will affect this

Reply:
/discover/multibuy should already be covered by the deep linking. This is just to transform a desktop link `/platform/:app_id` to the LLM `/discover/:app_id`